src/components/UI/DialogBox/DialogBox.module.css

```diff
@@ -49,13 +52,13 @@
 .DialogTitleCenter {
   composes: DialogTitle;
   text-align: center;
-  max-width: 395px;
+  max-width: 100%;
```

Comment:
Let's not change the max width. We have assigned a maximum width of the dialog box title and it should not be greater than 395px.

Reply:
When we use `px` we are adjusting the width exactly in the size `395px` meaning that it is static. But % allows readjusting depending on the browser and content.  Should I be using the equivalent of 395px to the `%` ?

Reply:
No. max-width property doesn't mean it will be set to this width. It means that the width can be extended to only 395px and not beyond that, and that is the behaviour we want.